From d890b363eb000d964750bf4ae6076c5d47b29168 Mon Sep 17 00:00:00 2001
From: Chris Burr <chrisburr@users.noreply.github.com>
Date: Sun, 28 Jun 2020 10:22:58 +0200
Subject: [PATCH] Fix typo in evtgen build configuration

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 68653bc..4878ab3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -60,7 +60,7 @@ if(EvtGen_FOUND)
   message(STATUS "Will attempt to link against EvtGen")
   set(EVTGEN    EvtGen::EvtGen)
   set(EVTGENEXT EvtGen::EvtGenExternal)
-  set(CMAKE_CXX_FLAGS    "${CMAKE_CXX_FLAGS} -DRAPID_EVTGEN -unresolved-symbols=ignore-in-shared-libs")
+  set(CMAKE_CXX_FLAGS    "${CMAKE_CXX_FLAGS} -DRAPID_EVTGEN --unresolved-symbols=ignore-in-shared-libs")
   set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 elseif(DEFINED ENV{EVTGEN_ROOT})
   message(STATUS "EVTGEN_ROOT is set: ${EVTGEN_ROOT}")
